### PR TITLE
LP-2588 Redirection and Textual Issues on ADG Platform

### PR DESCRIPTION
**Pull request by annas-usman**
[Ticket](https://philanthropyu.atlassian.net/browse/LP-2588)
[adg-edx-theme](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/170)

### CODE REVIEW

File: openedx/adg/lms/registration_extension/constants.py [hunk 1]
```diff
@@ -67,3 +67,6 @@
         ('Unaizah', _('Unaizah'))
     ]
 }
+
+REQUIRED_FIELD_CITY_MSG = _(u"Please select your City.")
+REQUIRED_FIELD_COMPANY_MSG = _(u"Please select your Company.")
```
Comment (asadali145) [inline openedx/adg/lms/registration_extension/constants.py:72]: Please use single quotes.

**Review by asadali145 — APPROVED** [review]
A small change, Rest LGTM

**Comment by annas-usman** [comment]
The code quality action is failing due to the unsuccessful cloning of msp assessment x-block

**Review by emraan004 — APPROVED** [review]
LGTM

**Pull request merged**